src/qibolab/instruments/qm/components/configs.py

```diff
@@ -8,16 +10,17 @@
 ]
 
 
-@dataclass(frozen=True)
 class OpxOutputConfig(DcConfig):
     """DC channel config using QM OPX+."""
 
+    kind: Literal["opx-output"] = "opx-output"
```

Comment:
These (and the equivalent for `QmAcquisitionConfig` a few lines below, are not used right now, as they are not registered in the `Config`. We could try the fix proposed in https://github.com/qiboteam/qibolab/pull/971#discussion_r1719986792 but since moving the `Config` may break some imports, maybe it is better to do in a different PR?

Reply:
I agree about the different PR, especially because you already managed to run protocols.